README.rst

```diff
@@ -4,9 +4,6 @@ URLExtract
 URLExtract is python class for collecting (extracting) URLs from given
 text based on locating TLD.
 
-.. image:: https://img.shields.io/travis/lipoja/URLExtract/master.svg
```

Comment:
Can we replace it with 
`https://img.shields.io/github/workflow/status/lipoja/URLExtract/Upload%20Python%20Package `
instead of removing it completely?

Reply:
Hi @lipoja what will be `target`?

Reply:
`https://github.com/lipoja/URLExtract/actions/workflows/python-publish.yml`

Reply:
Added @lipoja